package.json

```diff
@@ -20,12 +20,10 @@
   },
   "homepage": "https://github.com/FranckFreiburger/vue-pdf#readme",
   "dependencies": {
+    "@themainstage/pdfjs-dist": "^2.8.335-chromefix",
```

Comment:
What's the deal with this version instead of the official one?

Reply:
yeah, sorry, I started down this rabbit-hole trying to fix how pdf.js deals with large files in chrome -- see https://github.com/mozilla/pdf.js/issues/13452 if you're interested in the gory details.  They weren't interested in the slightest in trying to work around the issue, so I ended up forking just about everything.

If you're interested in merging this branch to pull in the upgrade to the latest version, let me know and I'll change the deps back, this should all work with the mainline pdfjs

Reply:
I don't have any right on this repo but I hope the author will consider merging this as there are quite a few issues opened here lately that can be solved with your fix (related to PDF.js renaming `es5` to legacy).